include/cinder/PolyLine.h

```diff
@@ -54,10 +54,11 @@ class PolyLineT {
 	T			getPosition( float t ) const;
 	T			getDerivative( float t ) const;
 
-	void		scale( const T &scaleFactor, T scaleCenter = T() );
-	void		offset( const T &offsetBy );
-	void		reverse();
-	
+	void			scale( const T &scaleFactor, T scaleCenter = T() );
+	PolyLineT<T>	scaled( const T &scaleFactor, T scaleCenter = T() ) const;
+	void			offset( const T &offsetBy );
+	PolyLineT<T>	offsetted( const T &offsetBy ) const;
+	void			reverse();
```

Comment:
I guess GitHub uses 8 spaces per tab? I'm using 4 per the [Contributing guidelines](https://github.com/cinder/Cinder/blob/master/CONTRIBUTING.md).